packages/main/src/Select.ts

```diff
@@ -349,6 +349,9 @@ class Select extends UI5Element implements IFormElement {
 	onBeforeRendering() {
 		this._syncSelection();
 		this._enableFormSupport();
+
+		const selectedIconCount = this.selectedOptionIcon ? 0 : 1;
```

Comment:
Is this correctly assigned to 0 when there is an icon in the selected option, or should it be the other way around?
I'm not sure about the expected behaviour and can't tell the difference with either of the values.